kafka/src/test/resources/HydraMetadataTopic.avsc

```diff
@@ -20,20 +20,25 @@
       "name": "subject",
       "type": "string"
     },
+    {
+      "name": "deprecated",
+      "type": ["null", "boolean"],
+      "default": null
+    },
     {
       "name": "streamType",
       "type": {
         "name": "streamTypeEnum",
         "type": "enum",
-        "symbols": ["Notification", "CurrentState", "History"]
+        "symbols": ["Notification", "CurrentState", "History", "Telemetry"]
       }
     },
     {
       "name": "dataClassification",
       "type": {
         "name": "dataClassificationEnum",
         "type": "enum",
-        "symbols": ["Public", "InternalUseOnly", "ConfidentialPII", "ConfidentialFinancial"]
```

Comment:
is there a problem with removing a symbol? what about existing metadata payload that have specified ConfidentialFinancial?